### DISCUSSION TITLE
Request contains a invalid argument when making request to queue

### DISCUSSION THREAD

**Discussion by robbyc73** [stackkit/laravel-google-cloud-tasks-queue#98] [Q&A] [answered]
Have configured my google application credentials and stackkit cloud tasks environment variables. Whenever i try to dispatch a job to configured google task queue i get the following:

`"{\n    \"message\": \"Request contains an invalid argument.\",\n    \"code\": 3,\n    \"status\": \"INVALID_ARGUMENT\",\n    \"details\": []\n}",`

Given there is not much in the response to go on other than something in request is invalid any help would be appreciated. I'll add some logging to try and get what is being sent through in request which is casuing issue

**Comment by marickvantuil** [comment, marked as answer]
That's indeed a not super helpful message! Some things I read only:

- Is the queue and the app (if running on app engine/cloud run/cloud function) in the same region?
- Service account email is correct?
- Is the task scheduled?
